mmv1/third_party/terraform/services/compute/resource_compute_security_policy.go.tmpl

```diff
@@ -701,8 +701,20 @@ func ResourceComputeSecurityPolicy() *schema.Resource {
 					},
 				},
 			},
+			"labels": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"label_fingerprint": {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
`label_fingerprint` should not be optional. It should not be provided by the user in the Terraform configuration.